src/d3a/models/area/redis_external_connection.py

```diff
@@ -70,7 +70,22 @@ def channel_unregister_callback(self, payload):
     def sub_to_area_event(self):
         channel = f"{self.area.slug}/register_participant"
         channel_unregister = f"{self.area.slug}/unregister_participant"
+        channel_market_stats = f"{self.area.slug}/market_stats"
 
         self.pubsub.subscribe(**{channel: self.channel_register_callback,
-                                 channel_unregister: self.channel_unregister_callback})
+                                 channel_unregister: self.channel_unregister_callback,
+                                 channel_market_stats: self.market_stats_callback})
         self.pubsub.run_in_thread(daemon=True)
+
+    def market_stats_callback(self, payload):
+        market_stats_respsonse_channel = f"{self.area.slug}/market_stats/response"
+        payload_data = json.loads(payload["data"])
+        ret_val = {"status": "ready",
+                   "market_stats":
+                       self.area.stats.get_market_price_stats(payload_data["market_slots"])}
+        self.redis_db.publish(market_stats_respsonse_channel, json.dumps(ret_val))
+
+    # def sub_to_market_stats(self):
+    #     channel = f"{self.area.slug}/market_stats"
+    #     self.pubsub.subscribe(**{channel: self.market_stats_callback})
+    #     self.pubsub.run_in_thread(daemon=True)
```

Comment:
Are you planning to use it somewhere?

Reply:
No, remnant. Already removed it locally.